modules/dnn/CMakeLists.txt

```diff
@@ -6,16 +6,36 @@ if(DEFINED BUILD_opencv_dnn AND NOT BUILD_opencv_dnn)
   return()
 endif()
 
-include(${OpenCV_SOURCE_DIR}/cmake/OpenCVFindLibProtobuf.cmake)
-if(NOT Protobuf_FOUND)
-  ocv_module_disable(opencv_dnn)
+ocv_option(${the_module}_USE_PROTOBUF "Force use of Protobuf library instead OpenCV's protobuf parser" OFF)
```

Comment:
This name is "_USE_PROTOBUF" because "the_module" is not defined here (before `ocv_add_module()`).

Reply:
Replaced under the `ocv_add_module`.

May I ask you to show version of system protobuf to reproduce it?

Added text descriptors parsing. So now we generate `.pb` from `.proto` using protoc. Then print it as text. I've found only one way to use python:
```python
from google.protobuf import descriptor_pb2

msg = descriptor_pb2.FileDescriptorSet()

with open('/path/to/example.pb', 'rb') as f:
    msg.ParseFromString(f.read())
    print msg
```